index.d.ts

```diff
@@ -50,7 +50,8 @@ export class DDWAF {
     exclusions?: diagnosticsInfo,
     rules_override?: diagnosticsInfo,
     rules_data?: diagnosticsInfo,
-    processors?: diagnosticsInfo
+    processors?: diagnosticsInfo,
+    actions?: diagnosticsInfo
```

Comment:
That is not what the test says:
![image](https://github.com/DataDog/dd-native-appsec-js/assets/70517128/1a9d85ab-c766-4e1a-8c48-adcad79b2deb)

Reply:
Actually, typings were wrong - not just for actions.

I've fixed them in this PR.